usr/lib/tik/modules/pre/20-mig

```diff
@@ -10,7 +10,7 @@ fi
 
 if [ ! -z "$(ls -A ${mig_dir})" ]; then
     log "existing backup found"
-    zenity --question --no-wrap --cancel-label="No, Delete Backup" --title="Existing user backup detected" --text="These users can be restored to the new installation\n\nWould you like to use this backup?"
+    d --question --no-wrap --cancel-label="No, Delete Backup" --title="Existing user backup detected" --text="These users can be restored to the new installation\n\nWould you like to use this backup?"
```

Comment:
Same as the similar comment in 15-encrypt, needs a `d-opt` or similar function because we're relying on the non-0 return code to inform us that the backup needs to be deleted, not that we want to stop the install (the default behaviour of `d`)